### DISCUSSION TITLE
整合数据库的docker版本什么时候能出呢

### DISCUSSION THREAD

**Discussion by darkelfzdy** [lobehub/lobehub#3337] [General Question] [answered]
整合数据库的docker版本什么时候能出呢？望眼欲穿

**Comment by arvinxx** [comment]
https://github.com/lobehub/lobe-chat/pull/2935 等这个 PR 合并吧，应该下周就能有了。还差一两个小问题等 @cy948 修掉就好

**Comment by darkelfzdy** [comment]
感谢答复，导入导出的同步太麻烦了

**Comment by xingpingcn** [comment]
是否会支持sqlite呢，我不知道社区的其他小伙伴会不会去实现这个需求

**Reply by arvinxx** [reply]
目前官方不计划支持，看社区小伙伴有没有类似诉求了

**Comment by darkelfzdy** [comment]
我看好像[#2935](https://github.com/lobehub/lobe-chat/pull/2935)已经合并进来了，docker镜像也更新了，是不是还需要更新一下docker的部署说明

**Reply by arvinxx** [reply]
莫急，还需要一个 PR 打成专门支持 db 的镜像

**Reply by arvinxx** [reply]
详见：


- https://github.com/lobehub/lobe-chat/issues/3386#issuecomment-2266643527
- https://github.com/lobehub/lobe-chat/pull/3387

**Comment by arvinxx** [comment, marked as answer]
server database docker 镜像已上线：https://github.com/lobehub/lobe-chat/issues/3391
